### PR TITLE
[android] Fix type of error message for building levels field

### LINKED ISSUE #4541: Wrong type of error message about buildings levels
During my tests in this PR #4423, i have forgotten to test with buildings levels field -> this field have old error message under field

https://github.com/organicmaps/organicmaps/blob/bab264f2b198af4fc0498abb55123aad85c7e991/android/src/app/organicmaps/editor/EditorFragment.java#L185

CC @biodranik @satvik2131

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Fix #4541

Before

After

I have tested, with others field it's work

### CODE REVIEW

File: android/src/app/organicmaps/util/UiUtils.java [hunk 1]
```diff
@@ -336,8 +336,14 @@ public static Uri getUriToResId(@NonNull Context context, @AnyRes int resId)
   public static void setInputError(@NonNull TextInputLayout layout, @StringRes int error)
   {
```
Comment (biodranik) [inline android/src/app/organicmaps/util/UiUtils.java:338]: This empty line can be removed

**Review by biodranik — APPROVED** [review]
LGTM if it works 😄

File: android/src/app/organicmaps/util/UiUtils.java (hunk 1 above)
Comment (Jean-BaptisteC) [reply android/src/app/organicmaps/util/UiUtils.java]: Done, thanks for your help :)

**Review by biodranik — APPROVED** [review]
Cool, thanks!

**Pull request merged**